components/cpx-url/README.md

```diff
@@ -2,32 +2,38 @@
 
 ## PURPOSE
 
-The `cpx-url` component is meant to be a declarative interface for getting and setting
-browser cookies and parsing them in various ways. JSON and JWT parsing is built in.
+The `cpx-url` component is meant to be a declarative interface for getting and
+setting browser cookies and parsing them in various ways. JSON and JWT parsing
+is built in.
 
 ## Installation
 
-* `npm install @chapeaux/cpx-url` or consume from a CDN via `@chapeaux/cpx-url`
-* include the `cpx-url.js` file on a page prior to any event tracking that should occur
+- `npm install @chapeaux/cpx-url` or consume from a CDN via `@chapeaux/cpx-url`
+- include the `cpx-url.js` file on a page prior to any event tracking that
+  should occur
 
 ## Usage
-* `<script type="module" src="path/to/@chapeaux/cpx-url/cpx-url.js"></script>`
-* `import { CPXWebSocket } from "path/to/@chapeaux/cpx-url/cpx-url.js";`
-*  ```html
-    <cpx-url></cpx-url>
 
-## Testing
+- `<script type="module" src="path/to/@chapeaux/cpx-url/cpx-url.js"></script>`
+- `import { CPXWebSocket } from "path/to/@chapeaux/cpx-url/cpx-url.js";`
```

Comment:
Should this import `CPXUrl`?

Reply:
Fixed in [4fe59ef](https://github.com/chapeaux/cpx-components/pull/21/commits/4fe59efda5865db469ae94e2ba4a16d2ce209905)